.github/workflows/pages.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll site to Pages
+
+on:
+  push:
+    branches:
+      - "main"
+      - "markdown-documentation"
```

Comment:
I'll do that as soon as we merge this PR, because if not, any changes in the docs folder won't be rendered.

I was planning on creating a tiny PR removing that branch after.